.github/workflows/integration-tests.yaml

```
@@ -0,0 +1,47 @@
+name: integration-tests
+
+env:
+  AWS_REGION: us-east-1
```

```suggestion
  AWS_REGION: eu-west-1
```
As noted in #1420 this is where integration tests need to be run